src/vm/wren_core.c

```diff
@@ -472,6 +482,11 @@ DEF_PRIMITIVE(fiber_yield1)
 
 DEF_PRIMITIVE(fn_new)
 {
+  RETURN_ERROR("Fn metaclass does not implement 'new()'.");
+}
```

Comment:
That would work for some of the classes, but not the ones that do need to define some methods in Wren, like String.